PureLayout/PureLayout/ALView+PureLayout.m

```diff
@@ -225,15 +225,22 @@ - (NSLayoutConstraint *)autoPinEdgeToSuperviewEdge:(ALEdge)edge withInset:(CGFlo
     if (edge != ALEdgeTop) {
         [constraints addObject:[self autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:insets.top]];
     }
-    if (edge != ALEdgeLeading && edge != ALEdgeLeft) {
```

Comment:
You seemed to have reversed the edges here? Ie. `ALEdgeTrailing` going to `ALEdgeLeading`? Same goes for the `ALEdgeRight` and `ALEdgeLeft`.

Reply:
I changed the `!=` to `==`. So before, it was `if the excluded edge is not leading, then pin to leading`, now it is `if the excluded edge is trailing, pin to leading`.  I changed it to `==` to make sure we're using either leading/trailing or right/left

Reply:
@toohotz

Reply:
@alyakan I follow now, it just seems a bit odd going from the non equality to equality and also the constraints originally are being added in a counterclockwise manner vs your changes now being clockwise so if I were used to making a reference to the second constraint out of the four that is returned before these changes, I would now end up with the opposite side constraint here.  @tomidelucca @mickeyreiss @jjksam  maybe you guys want to chime in here with your thoughts?

Reply:
Correct me if i'm wrong, but I believe that I still do follow the counter clockwise manner where I go by adding:
1. Top (if excluded edge is not top)
2. Leading/Left (if excluded edge is trailing/right)
3. Bottom (if excluded edge is not bottom)
4. Trailing/Right (if excluded edge is leading/left)

As for why I went to equality instead of inequality, it's just to simplify the if conditions because otherwise it would have been nested:
```
if (edge != ALEdgeLeading) {
    if (edge == ALEdgeRight) {
        // Add left edge
    } else {
       // Add leading edge
    }
}
```

Reply:
My mistake, I was reading the conditionals with the assumption of the old code logic, makes sense now and avoiding the nested conditionals as well. 👍🏾